app/code/Magento/Ui/TemplateEngine/Xhtml/Result.php

```diff
@@ -116,8 +124,11 @@ public function __toString()
             $this->appendLayoutConfiguration();
             $result = $this->compiler->postprocessing($this->template->__toString());
         } catch (\Throwable $e) {
-            $this->logger->critical($e->getMessage());
+            $this->logger->critical($e->getMessage(), ['exception' => $e->getTrace()]);
```

Comment:
can't we write while exception to log, like `$this->logger->critical($e)`?

Reply:
yes it will be better

Reply:
done